AdyenActions/UI/View Controllers/QR Code/QRCodeView.swift

```diff
@@ -130,6 +179,30 @@ internal final class PixView: UIView, Localizable, AdyenObserver {
         return instructionLabel
     }()
     
+    private lazy var qrCodeImageView: UIImageView = {
+        let data = model.action.qrCodeData.data(using: String.Encoding.ascii)
+        if let filter = CIFilter(name: "CIQRCodeGenerator") {
```

Comment:
It's better to return the `UIImageView()` in a guard and return the actual object outside of the guard to make it more prominent

Reply:
Sure :)